a1/src/main/java/ro/utcn/sd/it/a1/Controller/ConsoleController.java

```diff
@@ -0,0 +1,131 @@
+package ro.utcn.sd.it.a1.Controller;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.stereotype.Component;
+import ro.utcn.sd.it.a1.exception.QuestionNotFoundException;
+import ro.utcn.sd.it.a1.exception.UserNotFoundException;
+import ro.utcn.sd.it.a1.model.Question;
+import ro.utcn.sd.it.a1.model.Tag;
+import ro.utcn.sd.it.a1.model.User;
+import ro.utcn.sd.it.a1.service.QuestionManagementSystem;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.sql.Timestamp;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Optional;
+import java.util.Scanner;
+@Component
+@RequiredArgsConstructor
+public class ConsoleController implements CommandLineRunner {
+    private final Scanner scanner=new Scanner(System.in);
+    BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+   private User user = null;
+   private Tag tag=null;
+
+    private final QuestionManagementSystem qms;
+
+
+    @Override
+    public void run(String... args)  {
+        print("Welcome!");
+        boolean done=false;
+        while(!done){
+            print("Enter a command: ");
+            String command=scanner.next().trim();
```

Comment:
I strongly suggest that you get used to using the code formatter of your IDE. Usually developers use it as a reflex once they finish writing a method / block of code.  https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html